src/ui-shell/sidenav-header.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, Input } from "@angular/core";
+import { ListItem } from "./../dropdown/dropdown.module";
+import { I18n } from "./../i18n/i18n.module";
+
+@Component({
+	selector: "ibm-sidenav-header",
+	template: `
+		<header class="bx--side-nav__header">
+			<div class="bx--side-nav__icon">
+				<ng-content select="[icon]"></ng-content>
+			</div>
+			<div class="bx--side-nav__details">
+				<h2 class="bx--side-nav__title" [title]="title">{{title}}</h2>
+				<div class="bx--side-nav__switcher">
+					<label class="bx--assistive-text" for="switcherId">
+						{{i18n.get('UI_SHELL.SIDE_NAV.SWITCHER') | async}}
+					</label>
+					<select id="switcherId" class="bx--side-nav__select">
```

Comment:
Looks like it needs something like

```suggestion
					<select [id]="switcherId" class="bx--side-nav__select">
```

or

```suggestion
					<select [attr.id]="switcherId" class="bx--side-nav__select">
```

Reply:
`[id]` definitely just works